### ISSUE TITLE
CP-ABE with hidden access policy

### ISSUE THREAD

**Issue by zhangzhishun** [fentec-project/gofe#51]
Is there any implementation of CP-ABE encryption algorithm with hidden access policy? thanks

**Comment by tilenmarc** [comment]
Hi,
unfortunate not. Are you referring to a particular paper?
If you will be working on this, we are open to Pull Requests :)

**Comment by zhangzhishun** [comment]
> Hi, unfortunate not. Are you referring to a particular paper? If you will be working on this, we are open to Pull Requests :)

Yes, like this paper: “Reliable Policy Updating under Efficient Policy-Hidden Fine-grained Access Control Framework in the Cloud”
I am looking for the relevant implementation, I did not implement this part of the code
